lib/utils/html.js

```diff
@@ -10,3 +10,12 @@ export function escapeHTML(str) {
 		str.toString().replace(/[&"<>]/g, m => escapeLookups[m]) :
 		null;
 }
+
+/**
+ * removes all HTML tags and replaces all whitespaces with a single space
+ * @param {string} str The string to strip tags from
+ * @return {string} The stripped string or null if str is null
+ */
+export function stripTags(str) {
+	return str ? str.toString().replace(/<[^>]*?>/g, ' ').replace(/\s+/g, ' ') : null;
```

Comment:
Can you replace this with a third-party vendor function like `$(str).text()`? DIYing this will probably lead to obscure bugs.